### DISCUSSION TITLE
Findability of Resolutions within Issues

### DISCUSSION THREAD

**Discussion by mohits** [jruby/jruby#7066] [General]
I recently saw issue #7064 and @headius posted a solution there and I had this thought that we need to do something about solutions like these which are closed by requesting the user to do something specific. When I went back to get the issue number, I noticed this comment: https://github.com/jruby/jruby/issues/7064#issuecomment-1029033456 and Charles had added this:

> Perhaps we just need to SEO and get some visibility for this situation.

I'm thinking how we should do this. I searched for the exact issue with JRuby in google and it didn't throw up this issue or anything relevant. Adding github to the search term (which normal people don't do) helped a bit but not yet this issue.

I realised while looking at my blog posts that some of my posts get discovered because I have the exact error message that I was shown in the body of the post. This helps when someone does a blind copy and paste of the error (as I do) into Google to find a solution.

Should we do something like this?

**Comment by headius** [comment]
This could be a solution for issues that turn out to be a user problem or at least fixable by the user. Perhaps such issues that we mark as invalid should actually be moved into a discussion so they will be more searchable?

I actually ran into a similar problem when @byteit101 pointed out that the JRuby wiki is also not being indexed, and is therefore not searchable, using Google. GitHub appears to have some confusing policies in place for which project assets are crawlable and which ones are not. In the case of the wiki, we at least needed to turn off public edits, which is certainly not ideal but better than not having the wiki indexed at all.

I do frequently see JRuby issues show up in Google searches but I am not sure whether this is consistent. I don't know what the policy is for allowing search engines to crawl issues and discussions.

My hope is that issues *are* being indexed but #7064 is too new to have been crawled yet. Moving invalid issues to discussions with a marked answer might be a nice way to improve their reach, though.

Thoughts on this @enebo?

**Reply by mohits** [reply]
Since you and @enebo handle many/ most of the issues, do you think it's viable to add a tag to these issues when they are being closed (e.g. [Add to Wiki] or something) so that one of us others who can't contribute directly to the JRuby codebase could help with some of those items once there is a procedure that we can follow?

**Comment by headius** [comment]
A "needs documentation" label might work, but we'd need to periodically scan for such issues. Moving to a discussion might be simpler, if discussions are being found by search engines.

**Reply by enebo** [reply]
A documentation label might be the right move here if we want to close but let people know they can update our wiki with this info (in addition to seeing the closed issue in searches).  I don't think moving closed issues to a discussion is the right thing.  I do frequently see JRuby issues show up in searches.  I think issues is regularly getting indexed.  The wiki not getting indexed is new to me but once we have that indexed then I feel like we should move things which should be documented to the wiki.
